src/components/common/Dialogs/ManageFundsDialog/ManageFundsDialog.tsx

```diff
@@ -0,0 +1,227 @@
+  //   ? colony.status?.nativeToken?.unlockable
+  //   : hasRoot(allUserRoles) && colony.status?.nativeToken?.unlockable;
+
+  const canManageTokens = hasRoot(allUserRoles);
+
+  const items = [
+    {
+      title: MSG.transferFundsTitle,
+      description: MSG.transferFundsDescription,
+      icon: 'emoji-world-globe',
+      permissionRequired: false, // !canMoveFunds || isVotingExtensionEnabled,
+      permissionInfoText: MSG.permissionsListText,
+      permissionInfoTextValues: {
+        permissionsList: <FormattedMessage {...MSG.paymentPermissionsList} />,
+      },
+      onClick: () => callStep(nextStepTransferFunds),
+      dataTest: 'transferFundsDialogIndexItem',
+    },
+    {
+      title: MSG.mintTokensTitle,
+      description: MSG.mintTokensDescription,
+      icon: 'emoji-seed-sprout',
+      permissionRequired: false, // !canUserMintNativeToken,
+      permissionInfoText: MSG.permissionsListText,
+      permissionInfoTextValues: {
+        permissionsList: (
+          <FormattedMessage {...MSG.mintTokensPermissionsList} />
+        ),
+      },
+      onClick: () => callStep(nextStepMintTokens),
+      dataTest: 'mintTokensDialogItem',
+    },
+    {
+      title: MSG.manageTokensTitle,
+      description: MSG.manageTokensDescription,
+      icon: 'emoji-pen',
+      permissionRequired: !canManageTokens, // || isVotingExtensionEnabled
+      permissionInfoText: MSG.permissionsListText,
+      permissionInfoTextValues: {
+        permissionsList: (
+          <FormattedMessage {...MSG.manageTokensPermissionsList} />
+        ),
+      },
+      onClick: () => callStep(nextStepManageTokens),
+      dataTest: 'manageTokensDialogItem',
+    },
+    {
+      title: MSG.rewardPayoutTitle,
+      description: MSG.rewardPayoutDescription,
+      icon: 'emoji-piggy-bank',
+      comingSoon: true,
+    },
+    {
+      title: MSG.rewardsTitle,
+      description: MSG.rewardsDescription,
+      icon: 'emoji-medal',
+      comingSoon: true,
+    },
+    {
+      title: MSG.unlockTokensTitle,
+      description: MSG.unlockTokensDescription,
+      icon: 'emoji-padlock',
+      onClick: () => callStep(nextStepUnlockToken),
+      permissionRequired: false, // !canUserUnlockNativeToken,
+      permissionInfoText: MSG.permissionsListText,
+      permissionInfoTextValues: {
+        permissionsList: (
+          <FormattedMessage {...MSG.manageTokensPermissionsList} />
+        ),
+      },
+      dataTest: 'unlockTokenDialogIndexItem',
+    },
+  ];
+  const filteredItems =
+    colony?.status?.nativeToken?.mintable &&
+    colony?.status?.nativeToken?.unlockable
+      ? items
+      : items.filter(({ icon }) => {
+          if (
+            icon === 'emoji-padlock' &&
+            !colony?.status?.nativeToken?.unlockable
+          ) {
+            return true; // false
+          }
+          return true; // !(
+          //   icon === 'emoji-seed-sprout' &&
+          //   !colony.status?.nativeToken?.mintable
+          // );
```

Comment:
These comments left here without any TODOs/follow-up issues are a bit unclear

Reply:
I'll add a comment, but this is commented out because `colony.status`, as far as I know, is not implemented yet so I've decided to put these `true` here so that we can at least get to the "Mint tokens" and "Unlock tokens" dialogs. Otherwise, with the commented-out code those menu items would always be hidden.